docs/guide.md

```diff
@@ -0,0 +1,69 @@
+
+# Guide to the Dafny Core Library
+
+_THIS DOCUMENT IS A DRAFT UNDER ACTIVE DEBATE AND EDITING_
+
+A programming language library encapsulates a useful set of basic
+functionality that programmers can then incorporate in their custom
+program without having to rewrite, debug, or verify.
```

Comment:
Feels obvious for the target audience, I'd omit personally but open to suggestion from other reviewers.

Reply:
Removed. Boilerplate introduction.